### ISSUE TITLE
Next Up not displaying "Watchlisted" shows from Progress

### ISSUE THREAD

**Issue by metaltermite** [nixgates/plugin.video.seren#184]
There's an option in the [Trakt settings](https://trak.tv/settings) to `Include Watchlisted` shows in your Watched Progress (this includes any 0% progress shows). Eg. I have this option enabled, and have several shows with 0% progress on my Watchlist that correctly appear in my Watched Progress.

![Annotation 2019-10-25 010530](https://user-images.githubusercontent.com/10378599/67544651-53b67480-f6c4-11e9-9a5a-0a2b17c233cf.png)

Seren's Next Up is not displaying any Watchlisted shows that have 0% progress.

TLDR Next Up isn't mirroring my Watched Progress (https://trakt.tv/users/***/progress). It's failing to include the 0% progress Watchlisted shows.

**Comment by echo6ix** [comment]
I've noticed Seren is also not displaying items where the last aired episode of a show was watched but the show does not have 100% progress, eg. shows that are watched out of order, where the last aired episode was watched.

**Comment by drinfernoo** [comment]
@echo6ix That is by design. If the last episode of a show is watched, Seren (and Trakt) stops showing it in Next Up.

**Comment by echo6ix** [comment]
@drinfernoo I am confused as to why those shows are appearing in my Trakt's Dashboard Next Up feed as per the attached screen shot:

![next up](https://user-images.githubusercontent.com/37937507/67719337-0712b800-f9a8-11e9-87d8-305c5f70ce7a.png)

Nova is only 88% complete, last episode watched, an episode in the middle of the progress is missing. Still displayed in the Next Up feed on the dashboard, but not in Seren.

**Comment by nixgates** [comment]
@echo6ix Next Up is driven by the Trakt Sync Database to speed up the efficiency of the list.

There will be slight differences between the trakt list and your own.

The list within Seren is driven by watched items only, if you would like to see something in the list. I would recommend completing an episode item at least once. This will populate the next episodes for you as they are released.
